libgnucash/core-utils/gnc-filepath-utils.cpp

```diff
@@ -543,6 +543,33 @@ get_userdata_home(void)
     return userdata_home;
 }
 
+static bfs::path
+get_userconfig_home(void)
+{
+    gchar *config_dir = NULL;
+    auto userconfig_home = bfs::path();
+
+#ifdef G_OS_WIN32
+    config_dir = win32_get_userdata_home ();
+#elif defined MAC_INTEGRATION
+    config_dir = quarz_get_userdata_home ();
+#endif
+
+    /* On Windows and Macs the data directory is used, for Linux
+       $HOME/.config is used */
+    if (config_dir)
+    {
+        userconfig_home = config_dir;
+        g_free(config_dir);
+    }
+    else
+        userconfig_home = g_get_user_config_dir();
+
+    userconfig_home = userconfig_home / PACKAGE;
```

Comment:
Unfortunately this is the wrong spot to add the PACKAGE part of the path.
userconfig_home is supposed to be the equivalent of $HOME/.config
**gnc**_userconfig_home is intended to be $HOME/.config/gnucash

I am picky about this because I want userconfig and userdata to follow the same logic.